data/index.md

```diff
@@ -15,10 +15,10 @@ The Handbook consists of two main parts:
 
 ## Guides and Tutorials
 
+* [Walk-through of some Data Wrangling basic tasks]({{site.baseurl}}/data/tutorial/)
```

Comment:
why not just `(data-tutorial/)` - assuming that works (next items in list are like that)